.github/workflows/rust.yml

```diff
@@ -29,6 +29,12 @@ jobs:
         override: true
         components: rustfmt, clippy
 
+    - name: interface dependency
+      run: sudo /usr/share/rust/.cargo/bin/cargo install dbus-codegen
+
+    - name: generate interface
+      run: dbus-codegen-rust < durst/interface.xml > durst/src/interface.rs
```

Comment:
Shouldn't this be a part of cargo instead of our CI?

Reply:
I guess, this is the thing we want to have: https://doc.rust-lang.org/cargo/reference/build-scripts.html